### ISSUE TITLE
HONEST branded MPA's are excluded from searches

### ISSUE THREAD

**Issue by litepresence** [bitshares/bitshares-ui#3356]
**Describe the bug**
when I search for USD I get bitUSD and gateway USD assets... yet HONEST.USD is not shown.   Same is true of HONEST.BTC.

**Expected behavior**
HONEST.USD and HONEST.BTC both have liquid markets vs BTS and have both been active on the blockchain with 4+ feed producers for over a year.   Our brand should be included as part of this ecosystem. 

**Screenshots**
![image_2021-03-18_122140](https://user-images.githubusercontent.com/27382428/111660474-7dee6200-87e4-11eb-97ef-149626bed169.png)

**Comment by akledirs** [comment]
![image](https://user-images.githubusercontent.com/6587277/111889839-4276b200-89f5-11eb-8b8a-e644eed8f27e.png)
Do this!
![image](https://user-images.githubusercontent.com/6587277/111889873-8b2e6b00-89f5-11eb-8be1-16e0427fc47e.png)

**Comment by litepresence** [comment]
> Do this!

yes, I understand that you can make manual changes and add "HONEST" mpa's however I believe we're as much a part of the ecosystem as rudex, gdex, or xbtsx... and should be included in default searches without added user specified whitelisting.

Particularly HONEST.USD and HONEST.BTC have higher volume and have existed longer than many of the gateway tokens that are returned by default.
